cmd/image/import/import.go

```diff
@@ -1,16 +1,93 @@
 package _import
 
 import (
-	"fmt"
+	"github.com/ppc64le-cloud/pvsadm/pkg"
+	"github.com/ppc64le-cloud/pvsadm/pkg/client"
+	"github.com/ppc64le-cloud/pvsadm/pkg/utils"
 	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+	"os"
+	"strings"
 )
 
 var Cmd = &cobra.Command{
 	Use:   "import",
 	Short: "Import the image into PowerVS instances",
-	Long:  `Import the image into PowerVS instances`,
+	Long: `Import the image into PowerVS instances
+pvsadm image import --help for information
+
+# Set the API key or feed the --api-key commandline argument
+export IBMCLOUD_API_KEY=<IBM_CLOUD_API_KEY>
+
+Examples:
+
+# importing image from default region and using default storage type
+pvsadm image import -n upstream-core-lon04 -b <BUCKETNAME> --accesskey <ACCESSKEY> --secretkey <SECRETKEY> --s3-image-name rhel-83-10032020.ova.gz --image-name test-image
+
+# with user provided storage type and region
+pvsadm image import -n upstream-core-lon04 -b <BUCKETNAME> --accesskey <ACCESSKEY> --secretkey <SECRETKEY> -r <REGION> --disktype <STORAGETYPE> --s3-image-name rhel-83-10032020.ova.gz --image-name test-image
+
+# If user wants to specify the type of OS
+pvsadm image import -n upstream-core-lon04 -b <BUCKETNAME> --accesskey <ACCESSKEY> --secretkey <SECRETKEY> --s3-image-name rhel-83-10032020.ova.gz --image-name test-image --ostype <OSTYPE>
+`,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		fmt.Println("To Be Implemented...")
+		opt := pkg.ImageCMDOptions
+		//validate inputs
+		validOsType := []string{"aix", "ibmi", "redhat", "sles"}
+		validDiskType := []string{"tier3", "tier1"}
+		supportedRegions := []string{"us-east", "us-south", "eu-de", "eu-gb",
```

Comment:
this is not really needed if we are reading the region for the bucket directly

Reply:
and I don't want to maintain this list in the program and update each time when there is new storage bucket support, instead let PowerVS API throw an appropriate error